src/cli_shared/cli/client.rs

```diff
@@ -95,6 +97,10 @@ impl Default for Client {
             encrypt_keystore: true,
             metrics_address: FromStr::from_str("0.0.0.0:6116").unwrap(),
             rpc_address: SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), DEFAULT_PORT),
+            healthcheck_address: SocketAddr::new(
+                IpAddr::V4(Ipv4Addr::LOCALHOST),
```

Comment:
I'm assuming this is for local healthchecks (kubernetes or whatever)? Otherwise this should probably listen to `0.0.0.0`.

Reply:
Ahh, it's just a default.

Reply:
Yes, it's for local healthchecks, normally not meant to be exposed.